### DISCUSSION TITLE
GitHub Desktop can't access local repo

### DISCUSSION THREAD

**Discussion by bseigel** [community/community#139234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I use an unconventional arrangement to manage my repo. I map my dev and prod servers as network drives on my PC using a tool like WebDrive or Expandrive. I edit files in the DEV instance, then do the usual commits, and push up to GitHub.

I installed GitHub Desktop, and asked it to load my DEV repo, which is located at **I:\**. It gives me this for while:
![GitHubDesktop_nmGwlBFGEj](https://github.com/user-attachments/assets/5ba5b0a4-7e11-4c68-b528-292aa7c1f6ba)
and then says it can't access the repo. I've confirmed **.git** folder exists on my drive.
![GitHubDesktop_sd9FW3ibWK](https://github.com/user-attachments/assets/a81de703-f5a7-4b7c-99dd-221f8cc12fb4)

**Comment by ibra-kdbra** [comment]
GitHub Desktop maintains logs that can provide more insight into the issue:

Access Logs:
> In GitHub Desktop, go to "Help" > "Show Logs in Explorer".

> Review Recent Logs: Open the latest log files and look for error messages or warnings related to accessing the repository.

> Identify Specific Errors: Use these details to search for solutions or provide them.
